api/column_family.cc

```diff
@@ -939,15 +939,17 @@ void set_column_family(http_context& ctx, routes& r, sharded<db::system_keyspace
 
     ss::enable_auto_compaction.set(r, [&ctx](std::unique_ptr<http::request> req) {
         auto keyspace = validate_keyspace(ctx, req);
-        auto tables = parse_tables(keyspace, ctx, req->query_parameters, "cf");
+        auto table_infos = parse_table_infos(keyspace, ctx, req->query_parameters, "cf");
+        auto tables = table_infos | std::views::transform([] (auto& ti) { return ti.name; }) | std::ranges::to<std::vector>();
```

Comment:
If parse_tables() is just these two lines, then why not leave it as a convenience function with the same name it had, just this new definition using parse_table_infos, instead of duplicating these two lines many times?
The patch would be shorter, and there will be less code duplication (which your PR claims to reduce).

Reply:
Oh, I see now, these ugly "transform" lines will be gone later during this series....